fmod/src/steamaudio_fmod.cpp

```diff
@@ -512,3 +513,8 @@ void F_CALL iplFMODRemoveSource(IPLint32 handle)
 
     gSourceManager->removeSource(handle);
 }
+
+void F_CALL setGlobalHRTFDisabled(bool disabled) 
```

Comment:
For consistency with existing API functions, I would rename this to `iplFMODSetGlobalHRTFDisabled`.

Reply:
Done via bf260e754563f104c5d4eb0efed7acb3e340184f